IHP/IDE/CodeGen/ApplicationGenerator.hs

```diff
@@ -133,22 +131,30 @@ generateGenericApplication applicationName =
                 <> "</body>\n"
                 <> "|]\n"
                 <> "\n"
+                <> "stylesheets = do\n"
+                <> "    when (isDevelopment FrameworkConfig.environment) [hsx|\n"
+                <> "      <link rel=\"stylesheet\" href=\"/vendor/bootstrap.min.css\"/>\n"
+                <> "      <link rel=\"stylesheet\" href=\"/vendor/flatpickr.min.css\"/>\n"
+                <> "      <link rel=\"stylesheet\" href=\"/app.css\"/>\n"
```

Comment:
I think we need to add two more spaces here, as all other code is also indented with 4 spaces :)